Sources/MessagesViewController.swift

```diff
@@ -115,14 +121,12 @@ open class MessagesViewController: UIViewController {
        
         messagesCollectionView.translatesAutoresizingMaskIntoConstraints = false
 
-		let constraints: [NSLayoutConstraint] = [
-			messagesCollectionView.topAnchor.constraint(equalTo: view.topAnchor),
-			messagesCollectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-			messagesCollectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+        let top = messagesCollectionView.topAnchor.constraint(equalTo: topLayoutGuide.bottomAnchor)
```

Comment:
This may have to be changed to `constraint(equalTo: view.topAnchor, constant: topLayoutGuide.length)`